src/vcpkg/postbuildlint.cpp

```diff
@@ -1347,10 +1359,11 @@ namespace vcpkg
                 break;
                 case LinkageType::STATIC:
                 {
-                    auto dlls = release_dlls;
-                    dlls.insert(dlls.end(), debug_dlls.begin(), debug_dlls.end());
+                    auto& dlls = debug_dlls;
+                    dlls.insert(dlls.end(),
+                                std::make_move_iterator(release_dlls.begin()),
+                                std::make_move_iterator(release_dlls.end()));
```

Comment:
This moves out the contents of `release_dlls` which is used below on line 1374:

```cpp
error_count += check_crt_linkage_of_libs(fs, build_info, true, release_libs);
```

Reply:
That's release_libs, not release_dlls.